src/platform/macos/view.rs

```diff
@@ -403,6 +403,44 @@ fn get_characters(event: id) -> Option<String> {
     }
 }
 
+fn get_characters_ignoring_modifiers(event: id) -> Option<String> {
```

Comment:
It would be neater to just add an `ignore_modifiers` flag to `get_characters`, since that results in less unsafe code to audit.